1.10.9/alpine3.10/Dockerfile

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright 2019 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.10
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+
+LABEL io.astronomer.docker=true
+LABEL io.astronomer.docker.distro="alpine"
+LABEL io.astronomer.docker.module="airflow"
+LABEL io.astronomer.docker.component="airflow"
+LABEL io.astronomer.docker.airflow.version="1.10.9"
+
+ARG ORG="astronomer"
+ARG VERSION="1.10.9-1"
+ARG SUBMODULES="all, statsd, elasticsearch"
+ARG REPO_BRANCH=master
+
+ENV AIRFLOW_REPOSITORY="https://github.com/${ORG}/airflow"
+ENV AIRFLOW_MODULE="git+${AIRFLOW_REPOSITORY}@${VERSION}#egg=apache-airflow[${SUBMODULES}]"
+ENV AIRFLOW_HOME="/usr/local/airflow"
+ENV PYMSSQL_BUILD_WITH_BUNDLED_FREETDS=1
+ENV PYTHONPATH=${PYTHONPATH:+${PYTHONPATH}:}${AIRFLOW_HOME}
+ENV AIRFLOW_GPL_UNIDECODE="True"
+
+ARG ASTRONOMER_USER="astro"
+ARG ASTRONOMER_GROUP="astro"
+ENV ASTRONOMER_USER=${ASTRONOMER_USER}
+ENV ASTRONOMER_GROUP=${ASTRONOMER_GROUP}
+
+RUN addgroup -S ${ASTRONOMER_GROUP} \
+	&& adduser -S -G ${ASTRONOMER_GROUP} ${ASTRONOMER_USER}
+
+COPY apks/keys/humans@astronomer.io.rsa.pub /etc/apk/keys
+
+# Force pip to install these specific versions when ever it installs a module
+ENV PIP_CONSTRAINT=/usr/local/share/astronomer-pip-constraints.txt
+COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
+
+# Install packages
+RUN echo https://github.com/astronomer/ap-airflow/raw/${REPO_BRANCH}/1.10.9/alpine3.10/apks/repo >> /etc/apk/repositories \
+  && apk update \
+	&& apk add --no-cache --virtual .build-deps \
+		build-base \
+		cyrus-sasl-dev \
+		freetds-dev \
+		freetype-dev \
+		git \
+		krb5-dev \
+		libffi-dev \
+		libxml2-dev \
+		libxslt-dev \
+		linux-headers \
+		nodejs \
```

Comment:
This'll need updating when #41 is merged to follow that same pattern.

Reply:
Updated